plugins/usbdmx/EuroliteProFactory.h

```diff
@@ -35,22 +36,30 @@ namespace usbdmx {
  */
 class EuroliteProFactory : public BaseWidgetFactory<class EurolitePro> {
  public:
-  explicit EuroliteProFactory(ola::usb::LibUsbAdaptor *adaptor)
-      : BaseWidgetFactory<class EurolitePro>("EuroliteProFactory"),
-        m_adaptor(adaptor) {}
+  explicit EuroliteProFactory(ola::usb::LibUsbAdaptor *adaptor,
```

Comment:
Any reason this doesn't inherit from the BaseWidgetFactory here now?

Reply:
I'm afraid I don't understand what you mean here. EuroliteProFactory still inherits from BaseWidgetFactory, but I moved the constructor to the cpp file because the constructor now contains some logic. Is that what you mean?